region/brinstar/red/Red Brinstar Fireflea Room.json

```diff
@@ -522,9 +522,8 @@
       "link": [2, 1],
       "name": "X-Ray Access Ice Bridge",
       "requires": [
-        {"notable": "X-Ray Access Ice Bridge"},
         "canTrickyUseFrozenEnemies",
-        "canCarefulJump",
+        "canTrickyDodgeEnemies",
```

Comment:
This would keep this in Expert, but it would not be notable. It does require ice and 3 spikes, so you'll still typically go for the d-boost strat instead. This one is awkward though, because this can still be expected if the d-boost variant is turned off. So it could be kept notable if that's going to be a problem.